full-stack-tests/core/src/frontend/standalone/CachedDecorations.test.ts

```diff
@@ -0,0 +1,184 @@
+/*---------------------------------------------------------------------------------------------

```

Comment:
Looks like there was some fairly basic testing in Decorations.test.ts for cached decorations - it's great to have this very thorough test now.

Reply:
Most of it was already there, I just renamed the file and added test for my changes.

Reply:
@pmconne Separate files makes it clearer... nice.

Reply:
@pmconne Oh, looks like we don't have a Decorations.test.ts or anything which just has decorations tested.  I'm sure we do somewhere have more through decoration testing, right?

Reply:
See prev reply...I renamed `Decorations.test.ts` to `CachedDecorations.test.ts`, did some minor cleanup of existing tests, and added my new tests.

Reply:
I gotcha now - I initially looked at this file wrong.  Thanks for clearing me up.
Not for this PR, but maybe we could think about testing decorators in some kind of more automated way (rendering one, checking the pixels) like we do for mirukuru, etc.  Might catch some more stuff.

Reply:
More tests == more better.
Also for separate PR: tests like these ones that use a Viewport but don't care about a specific iModel should be moved to imodeljs-frontend and use a BlankConnection. The annoying part of that is we'd want to be able to use TestViewport.ts which is currently in core-full-stack-tests.